debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleConnectorTask.java

```diff
@@ -139,4 +164,100 @@ public void doStop() {
     protected Iterable<Field> getAllConfigurationFields() {
         return OracleConnectorConfig.ALL_FIELDS;
     }
+
+    private void validateRedoLogConfiguration() {
+        // Check whether the archive log is enabled.
+        final boolean archivelogMode = jdbcConnection.isArchiveLogMode();
+        if (!archivelogMode) {
+            throw new DebeziumException("The Oracle server is not configured to use a archive log LOG_MODE, which is "
+                    + "required for this connector to work properly. Change the Oracle configuration to use a "
+                    + "LOG_MODE=ARCHIVELOG and restart the connector.");
+        }
+    }
+
+    private boolean validateAndLoadDatabaseHistory(OracleConnectorConfig config, OraclePartition partition, OracleOffsetContext offset, OracleDatabaseSchema schema) {
```

Comment:
I understand we took this method as a cut-n-paste from MySQL; however, I dislike that we return a value from the method but the return value is never used so technically the method could simply have a `void` return value.